src/constants/layoutConstant.ts

```diff
@@ -0,0 +1,5 @@
+/**
```

Comment:
when we are already in a folder called "constants", we need not to suffix our files with "XConstant" anymore. These files can be just layout and theme